### ISSUE TITLE
Use different color for the keys in JSON

### ISSUE THREAD

**Issue by ffes** [rouge-ruby/rouge#819]
When you look at this simple JSON sample you see that the keys and the values use the same (string) highlighting, here on GitHub as well as in a Rouge generated HTML. Their behavior is the same.

```json
{
  "key": "value",
  "a_bit_longer_key": "a_much_longer_value"
}
```

My editor of choice vscode highlights the keys different:

![json-snippet](https://user-images.githubusercontent.com/3225656/32543198-3bbb9c80-c475-11e7-8c58-6f91786bd720.png)

It would be great if Rouge could generate something similar.
I think just giving the keys the class `.k` aka `Keyword` would do the trick.

**Comment by Gidgidonihah** [comment]
This used to be the case before 2.0.6 :/

https://github.com/jneen/rouge/issues/675

**Comment by pyrmont** [comment]
This was added via #1029. It was released in v3.7.0. Apologies for the lack of notice!
